app/services/message_employee_matcher.rb

```diff
@@ -0,0 +1,17 @@
+class MessageEmployeeMatcher
+  def initialize(message)
+    @message = message
+  end
+
+  def run
+    Employee.where(started_on: day_count.days.ago).pluck(:id)
```

Comment:
might make more sense to return `slack_username` here since we are using the `id` to find the object/slack username right after this...